source/full_shutdown.rst

```diff
@@ -0,0 +1,205 @@
+.. include:: vars.rst
+
+=======================
+Full Shutdown Procedure
+=======================
+
+In case a full shutdown of the system is required, we advise to use the
+following order:
+
+* Perform a graceful shutdown of all virtual machine instances
+* Stop Ceph (if applicable)
+* Put all nodes into maintenance mode in Bifrost
+* Shut down compute nodes
+* Shut down monitoring node
+* Shut down network nodes (if separate from controllers)
+* Shut down controllers
+* Shut down Ceph nodes (if applicable)
+* Shut down seed VM
+* Shut down Ansible control host
+
+Virtual Machines shutdown
+-------------------------
+
+Contact Openstack users to stop their virtual machines gracefuly, 
+If that is not possible shut down VMs using openstack CLI as admin user:
+
+.. code-block:: bash
+
+   for i in `openstack server list --all-projects -c ID -f value` ; \
+   do openstack server stop $i ; done
+
+
+.. ifconfig:: deployment['ceph_managed']
+
+   Stop Ceph
+   ---------
+   Procedure based on `Red Hat documentation <https://access.redhat.com/documentation/en-us/red_hat_ceph_storage/4/html/administration_guide/understanding-process-management-for-ceph#powering-down-and-rebooting-a-red-hat-ceph-storage-cluster_admin>`__ 
```

Comment:
If there's something equivalent in the community docs it would be better, but the closest I found was https://docs.ceph.com/en/latest/rados/operations/operating/ and it doesn't cover setting all the flags below.